Milestone 5-making calculator functional

ajkacca457 opened the pull request:
In this milestone I have:

- created events for all the buttons in ButtonPanel.
- created a function to display the result in the display panel.
- added additional styling to the calculator.
- passed click event through props.

uche-inyama reviewed (changes requested):
## Changes Required

### Hi @ajkacca457 :wave: 

You meet the requirements for this milestone. However, your deployed application is not running properly, it does not show the calculator. 

![snaoo](https://user-images.githubusercontent.com/46329537/106507907-fbc40b80-64cb-11eb-8ab1-0d8d1f175246.png)

* [ ] Also, in your readme, add the script for starting the application.

#### That's it make this updates, then submit another form for a code review.


### Have a great day :slightly_smiling_face: 

### ``` Uchechukwu Henry Inyama ```

ajkacca457 commented:
@uche-inyama  thank you for your review. Your review was very helpful in case of improving my project. Based on your review, I have fixed the live heroku link. I have added the "npm run start" command in the readMe file. Hopefully, this will meet the additional requirement.

Torres-ssf reviewed (approved):
### Milestone Approved :+1: 
Hi @ajkacca457 :hand: 
Great job on your project.
You may merge the PR.

Happy coding :computer: 
**Sergio Torres**
